mantidproject/mslice
# Subtract script generation bugfix

Fixes a bug in the generation of a script from a workspace created by subtracting two other workspaces.

**To test:**

Download the datafiles here:  [subtract_bug.zip](https://github.com/mantidproject/mslice/files/2989165/subtract_bug.zip)). Load both datafiles, and subtract one from the other using the `Subtract` button. Plot a slice or cut of the subtracted workspace, and then save a script file from the plot (`File`->`Generate Script`). Run the generated file (`import <file>`) and check that it recreates the plot you generated from. 

In previous versions the subtracted file is not reference properly and running the generated script will give an error.


Fixes #458
